### PR TITLE
Remove all the green checkmark ✔️ in the Content column #280

### PR DESCRIPTION
**Pull request by dcompanykrish**
Closes: #280

> *Be noted that 280 is the issue ID*

## What is the purpose of the change

>All green checkmarks ✔️ in the Content column have been removed.
The table has been updated to reflect the change.

### CODE REVIEW

**Review by tungbq — COMMENTED** [review]
Seems like you mixed 2 separated tasks into one:
image

We should keep 1 PR for each issue.
Could you please check and update this point?

**Comment by tungbq** [comment]
> Thank you @dcompanykrish But it seems like you mixed 2 separated tasks into one: image
> 
> We should keep 1 PR for each issue. Could you please check and update this point?

@dcompanykrish  Also, if you have some trouble with the Git usage, please refer to this guide https://github.com/tungbq/devops-basic/blob/main/CONTRIBUTING.md. Hope this help

File: topics/docker/README.md [hunk 1]
```diff
@@ -1,12 +0,0 @@
-# What is docker?
```
Comment (tungbq) [inline topics/docker/README.md:1]: Please don't remove this file, can you double check? Thanks

**Review by tungbq — CHANGES_REQUESTED** [review]

**Comment by dcompanykrish** [comment]
Hi @tungbq ,
I understand that we should not remove the file, and I have made the necessary changes to the PR to reflect this. 🔒

Please review the changes and let me know if you have any questions or concerns. 💬

**Review by tungbq — APPROVED** [review]
Great, thank you @dcompanykrish. Appreciate your great effort in this repo!
If you find this repository helpful, kindly consider showing your appreciation by giving it a star ⭐ Thanks! 💖

**Pull request merged**